src/PriceOracle.sol

```diff
@@ -95,7 +96,7 @@ contract PriceOracle is AccessControl {
             // redundancy may not be zero
             require(redundancy > 0, "unexpected zero");
 
-            // maximum extra redundancy is 4
+            // maximum considered extra redundancy is 4
```

Comment:
To me this is a bit strange that the comment here talks about `4` but then the actual code it checks if the redundancy is bigger than 8. I can think any of the following could be an improvement:
- Mentioning in the comment that above a certain value there is no need for more redundancy
- Maybe extract the value 8 to a constant and name it to something that is helpful in this context

Reply:
i have extracted the constants and tried to clarify this, hopefully it makes sense now. imo there is a pending task to fix conventions eg. capitalising comments and standardising use of a preceding underscore.